app/src/main/java/org/liberty/android/fantastischmemo/dao/MultipleChoiceDaoImpl.java

```diff
@@ -74,10 +74,8 @@ public MultipleChoiceCard getMultipleChoiceCard(long id) {
     public MultipleChoiceCard addMultipleChoiceCard(MultipleChoiceCard mcCard) {
         AnyMemoDBOpenHelper dbHelper = getHelper();
         dbHelper.insertMultipleChoiceCard(mcCard, dbHelper.getWritableDatabase());
-        if (mcCard != null) {
-            return mcCard;
-        }
-        return null;
+
+        return mcCard;
```

Comment:
Why does an add return a value? Shouldn't it be void?

Reply:
to be honest I think having it return the card that was added is meaningful. But I can change it to void

Reply:
So leave it. If u think its good I'm ok with that. Was just curious.